source/threads.tex

```diff
@@ -94,7 +94,7 @@
 The functions whose names end in \tcode{_for} take an argument that
 specifies a duration. These functions produce relative timeouts. Implementations
 should use a steady clock to measure time for these functions.\footnote{All
```

Comment:
```suggestion
should use a steady clock to measure time for these functions.%
```

Reply:
done, but why the trailing %?

Reply:
To avoid a space between the period and the footnote marker.